src/components/TimeseriesChartMeta/stories/TimeseriesChartMeta.stories.tsx

```diff
@@ -12,40 +14,49 @@ import full from './full.md';
 import hideElements from './hideElements.md';
 import predefinedPeriod from './predefinedPeriod.md';
 
-const setupMocks = () => {
-  setupTimeseriesChartMocks();
+const fakeClient: API = {
+  ...timeseriesChartFakeClient,
   // @ts-ignore
-  sdk.TimeSeries.retrieve = () => {
-    return new Promise(resolve => {
-      setTimeout(
-        () =>
-          resolve({
-            id: 8681821313339919,
-            name: 'IA_21PT1019.AlarmByte',
-            isString: false,
-            unit: 'bar',
-            metadata: {
-              tag: 'IA_21PT1019.AlarmByte',
-              scan: '1',
-              span: '100',
-              step: '1',
-              zero: '0',
-            },
-            assetId: 4965555138606429,
-            isStep: false,
-            description: '21PT1019.AlarmByte',
-          }),
-        1000
-      );
-    });
-  };
+  timeseries: {
+    retrieve: (): Promise<GetTimeSeriesMetadataDTO[]> => {
+      return new Promise(resolve => {
+        setTimeout(
+          () =>
+            resolve([
+              {
+                id: 8681821313339919,
+                name: 'IA_21PT1019.AlarmByte',
+                isString: false,
+                unit: 'bar',
+                metadata: {
+                  tag: 'IA_21PT1019.AlarmByte',
+                  scan: '1',
+                  span: '100',
+                  step: '1',
+                  zero: '0',
+                },
+                assetId: 4965555138606429,
+                isStep: false,
+                description: '21PT1019.AlarmByte',
+                createdTime: new Date(0),
+                lastUpdatedTime: new Date(0),
+              },
+            ]),
+          1000
+        );
+      });
+    },
+  },
 };
 
 storiesOf('TimeseriesChartMeta', module).add(
   'Full description',
   () => {
-    setupMocks();
-    return <TimeseriesChartMeta timeseriesId={123} />;
+    return (
+      <ClientSDKProvider client={fakeClient}>
```

Comment:
It's better to use a storybook decorator here:
```jsx
const clientSDKDecorator = (storyFn: any) => (
  <ClientSDKProvider client={fakeClient}>{storyFn()}</ClientSDKProvider>
);
```

and then
```jsx
storiesOf('AssetEventsPanel', module)
  .addDecorator(clientSDKDecorator)
  .add(
```

Reply:
Fixed